skfda/datasets/_real_datasets.py

```diff
@@ -1234,7 +1234,8 @@ def fetch_gait(
 
 if fetch_gait.__doc__ is not None:  # docstrings can be stripped off
     fetch_gait.__doc__ += _gait_template.format(
-        cite=":footcite:p:`ramsay+silverman_2005_introduction`",
+        cite="Ramsay and Silverman (chapter 1)"
```

Comment:
These are not being rendered well (compare them with MCO). I think a space is missing before `:footcite:` (and I am not sure if the trailing dot is necessary).

Reply:
Ok, you fixed `fetch_gait`, but you didn't fix `fetch_handwriting` (the comment refers to both).